src/cloudai/cli/handlers.py

```
@@ -281,12 +298,35 @@ def _check_installation(
+
+
 def handle_dry_run_and_run(args: argparse.Namespace) -> int:
     setup_result = _setup_system_and_scenario(args)
     if setup_result is None:
         return 1
     system, test_scenario, tests = setup_result
 
+    try:
+        validate_dse_env_params(test_scenario)
+    except TestScenarioParsingError as e:
+        logging.error(str(e))
```

Ruff (0.15.17)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)